python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -135,3 +138,247 @@ def create_ui_url(databricks_client, op_config):
         return "https://{host}/?o={workspace_id}#joblist".format(
             host=host, workspace_id=workspace_id
         )
+
+
+def create_databricks_run_now_op(
```

Comment:
Relevant discussion here btw: https://elementl-workspace.slack.com/archives/C04JNJCSS13/p1677526031428709

Reply:
Maybe worth a chat? I agree that this is an API that's hard to walk back and I'm a little nervous about it, but I also think it's a promising pattern.  

Maybe best to be conservative and leave out the runtime config for now?

Having two separate top-level APIs seems costly to me - I think many users will find the distinction immaterial and be deterred by needing to make a choice.

Reply:
I think the confusion comes from the underlying Databricks API call.

When using the `Run Now` API, you can specify a custom list of params to pass into the underlying Databricks Job. Otherwise, the default params of the Databricks Job will be used.

Similarly, when using the `Submit Run` Databricks API, you can specify the list of Databricks tasks you want to run, as well as the configuration of the cluster on which the tasks run. It is this cluster configuration that could be useful to configure at run time, but from talking to some users, cluster configuration is rarely manipulated at run time - if any changes are made to it, it would probably be a permanent change, meaning a code change would be required.

I can get rid of its config for now then, and wait until a user explicitly asks for this run-time configuration of their jobs.